chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/changerequest/presenter/PendingChangeRequestPresenter.java

```diff
@@ -0,0 +1,102 @@
+package gov.healthit.chpl.scheduler.job.changerequest.presenter;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.logging.log4j.Logger;
+
+import gov.healthit.chpl.changerequest.domain.ChangeRequest;
+import gov.healthit.chpl.domain.CertificationBody;
+import gov.healthit.chpl.util.DateUtil;
+
+public class PendingChangeRequestPresenter extends ChangeRequestCsvPresenter {
+    private static final String ACB_APPLICABLE_TEXT = "Applicable";
+    private static final String ACB_NOT_APPLICABLE_TEXT = "Not Applicable";
+    private static final long MILLIS_PER_DAY = 1000 * 60 * 60 * 24;
+
+    private List<String> acbNames;
+    private Date currentDate;
+
+    public PendingChangeRequestPresenter(Logger logger, List<String> acbNames) {
+       super(logger);
+       if (!CollectionUtils.isEmpty(acbNames)) {
+           this.acbNames = acbNames.stream().sorted().collect(Collectors.toList());
+       }
+       this.currentDate = new Date();
+    }
+
+    protected boolean isSupported(ChangeRequest data) {
+        return true;
```

Comment:
?

What is this?

Reply:
An implementation of an abstract method... the presenter can support some change requests (like just demographic ones, just attestation ones, or some other subset) or it can support them all, as the PendingChangeRequestPresenter does.